app/controllers/api/v2/hosts_controller.rb

```diff
@@ -265,8 +265,9 @@ def facts
 
       api :PUT, "/hosts/:id/rebuild_config", N_("Rebuild orchestration config")
       param :id, :identifier_dottable, :required => true
+      param :only, Array, :desc => N_("Limit rebuild steps")
```

Comment:
Can you give examples of values which are expected here? Users won't know if you expect IDs, names or something else. I think you should be able to call `rebuild_methods` directly and it will generate fine (this needs to be tested).